Refresh.GameServer/Types/Reviews/SerializedGameReview.cs

```diff
@@ -48,16 +49,30 @@ public class SerializedGameReview : IDataConvertableFrom<SerializedGameReview, G
     public int ThumbsDown { get; set; }
     
     [XmlElement("yourthumb")]
-    public RatingType YourThumb { get; set; }
+    public int YourThumb { get; set; }
 
     public static SerializedGameReview? FromOld(GameReview? review, DataContext dataContext)
     {
         if (review == null) 
             return null;
         
-        DatabaseRating reviewRating = dataContext.Database.GetRatingForReview(review);
+        DatabaseRating reviewRatings = dataContext.Database.GetRatingForReview(review);
+
+        bool isRated = dataContext.Database.ReviewRatingExistsByUser(dataContext.User!, review);
+        // neutral by default
+        RatingType userRatingType = RatingType.Neutral;
+        
+        // if it is rated by current user
+        // then get that rating and set YourThumb to that
+
+        if (isRated) {
+            // the review rating already exists hence the ! ignores
+            RateReviewRelation? reviewRating =
+                dataContext.Database.GetRateReviewRelationForReview(dataContext.User!, review);
+
+            userRatingType = reviewRating!.RatingType;
+        }
```

Comment:
This can be massively simplified to something like this:
```suggestion
        RatingType userRatingType = dataContext.Database.GetRateReviewRelationForReview(dataContext.User!, review)?.RatingType ?? RatingType.Neutral;
```